crates/bevy_ecs/src/storage/sparse_set.rs

```diff
@@ -447,6 +447,12 @@ impl<I: SparseSetIndex, V> SparseSet<I, V> {
         })
     }
 
+    pub fn clear(&mut self) {
+        self.dense.clear();
+        self.indices.clear();
+        self.sparse.clear();
+    }
+
```

Comment:
this seems unrelated?

Reply:
Oh that was a remnant from the old approach. I'll remove it.